src/test/java/org/prebid/server/bidder/vox/VoxBidderTest.java

```diff
@@ -0,0 +1,203 @@
+package org.prebid.server.bidder.vox;
+
+import com.iab.openrtb.request.BidRequest;
+import com.iab.openrtb.request.Imp;
+import com.iab.openrtb.response.Bid;
+import com.iab.openrtb.response.BidResponse;
+import com.iab.openrtb.response.SeatBid;
+import io.vertx.core.MultiMap;
+import io.vertx.core.http.HttpMethod;
+import lombok.SneakyThrows;
+import lombok.Value;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.prebid.server.VertxTest;
+import org.prebid.server.bidder.model.BidderBid;
+import org.prebid.server.bidder.model.BidderCall;
+import org.prebid.server.bidder.model.HttpRequest;
+import org.prebid.server.bidder.model.HttpResponse;
+import org.prebid.server.bidder.model.Result;
+import org.prebid.server.proto.openrtb.ext.ExtPrebid;
+
+import java.util.List;
+import java.util.function.Function;
+
+import static java.util.function.Function.identity;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+import static org.prebid.server.bidder.model.BidderError.badServerResponse;
+import static org.prebid.server.proto.openrtb.ext.response.BidType.audio;
+import static org.prebid.server.proto.openrtb.ext.response.BidType.banner;
+import static org.prebid.server.proto.openrtb.ext.response.BidType.video;
+import static org.prebid.server.proto.openrtb.ext.response.BidType.xNative;
+import static org.prebid.server.util.HttpUtil.ACCEPT_HEADER;
+import static org.prebid.server.util.HttpUtil.APPLICATION_JSON_CONTENT_TYPE;
+import static org.prebid.server.util.HttpUtil.CONTENT_TYPE_HEADER;
+import static org.prebid.server.util.HttpUtil.headers;
+import static org.springframework.util.MimeTypeUtils.APPLICATION_JSON_VALUE;
+
+class VoxBidderTest extends VertxTest {
+
+    private static final String ENDPOINT_URL = "https://test-url.com";
+    private static final String CURRENCY = "USD";
+
+    private VoxBidder sut;
+
+    @Test
+    public void shouldFailOnBidderCreation() {
+        assertThatIllegalArgumentException().isThrownBy(() -> new VoxBidder("invalid_url", jacksonMapper));
+    }
+
+    @Test //todo: not sure whether it's fine
```

Comment:
From core logic, it is not possible that a request without imp reaches the bidder.
Because imp has information, about what bidder should be reached

Reply:
do we need such kind of test? or you think it's confusing?

Reply:
I will remove